### ISSUE TITLE
Sort-AppFoldersByDependencies can't handle twice the same app in the baseFolder

### ISSUE THREAD

**Issue by Spycoclown** [microsoft/navcontainerhelper#1516]
Hello,

in the issue i reported a bug:
https://github.com/microsoft/navcontainerhelper/issues/1507

is just tested your fix, unfortunately, you fixed only AppFilesByDependencies and not AppFoldersByDependencies.

```
BcContainerHelper version 1.0.15
##[error]ForEach-Object : Item has already been added. Key in dictionary: '6a6d936f-1c1f-44c8-b3b5-06bcb744a2c2'  Key being 
added: '6a6d936f-1c1f-44c8-b3b5-06bcb744a2c2'
At C:\Program 
Files\WindowsPowerShell\Modules\BcContainerHelper\1.0.15\AppHandling\Sort-AppFoldersByDependencies.ps1:36 char:19
+     $appFolders | ForEach-Object {
+                   ~~~~~~~~~~~~~~~~
    + CategoryInfo          : OperationStopped: (:) [ForEach-Object], ArgumentException
    + FullyQualifiedErrorId : System.ArgumentException,Microsoft.PowerShell.Commands.ForEachObjectCommand
##[error]PowerShell exited with code '1'.
```

**Comment by freddydk** [comment]
Yeah - sorry, will apply the same fix in the next preview, sorry

**Comment by freddydk** [comment]
Fix shipped in 1.0.16
